cypress/integration/products/buy_a_product.js

```diff
@@ -4,7 +4,7 @@ import { HEADER_SELECTORS } from "../../elements/main-header/header-selectors";
 import { PRODUCTS_SELECTORS } from "../../elements/products/products-selectors";
 import { CHECKOUT_SELECTORS } from "../../elements/products/checkout-selectors";
 
-const randomWord = faker.random.words(2).replace(" ", "-");
```

Comment:
Should we rely on randomly-generated input for our end-to-end tests?

Reply:
@patrys please notice that I am using it for generating a fake email address. Do you have a better idea of how to handle it?

Reply:
I think we should be fine with using predefined addresses as long as those don't exist in the test database. Debugging tests with random input is a nightmare when they only fail one time out of ten.

Reply:
In this case, it is used only for a billing address (required field), the problem was that that previously email address had randomly spaces right now this situation is not gonna happened. I don't see a problem with fake emails here, but of course, we can change that to the same email which is used in login tests but in this scenario, I am gonna get emails every time when the test will run :x . (testers@saleor.io)

Reply:
Do we send confirmation to those emails? If so, they need to be valid emails or we're going to get in trouble with AWS.

Reply:
Ok, so I'll change that. FYI https://app.clickup.com/t/6uv31e